### DISCUSSION TITLE
Automatically redirected on callback page

### DISCUSSION THREAD

**Discussion by amirakbulut** [nuxt-community/auth-module#1521] [Support Requests]
(Using Auth0)

When logging in, I get redirected to my callback page, which automatically redirects me to the user dashboard. Why?

I need to run some logic before redirecting my user to the dashboard, so is there a way to do this manually? Here's my config:

```
 auth: {
  redirect: {
    login: '/auth/login',       // redirect if login is required
    logout: '/home',            // redirect after logout
    callback: '/callback',     // callback after logged in
    home: '/dashboard'          // redirect after logged in
  },
  localStorage: false,
  resetOnError: true,
  cookie: {
    prefix: 'auth.'
  },
  strategies: {
    local: false,
    auth0: {
      domain: process.env.AUTH0_DOMAIN,
      client_id: process.env.AUTH0_CLIENT_ID,
      audience: process.env.AUTH0_AUDIENCE,
      scope:"openid profile email",
    }
  }
},
```

**Comment by xiaofei-du** [comment]
From my understanding as a newbie, I think it is because `home` is set to `/dashboard`, when the user is redirected to the callback page, he/she is also logged in, and you set 
```
home: '/dashboard'          // redirect after logged in
```
like you commented, the user will be redirected to `home`.

If you set `home:  false`, the user will be redirected to the callback page and stop there.

**Reply by bmulholland** [reply]
See https://github.com/nuxt-community/auth-module/issues/1010
